content/en/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace.md

```diff
@@ -141,6 +141,15 @@ Error from server (Forbidden): error when creating "examples/admin/resource/quot
 pods "quota-mem-cpu-demo-2" is forbidden: exceeded quota: mem-cpu-demo,
 requested: requests.memory=700Mi,used: requests.memory=600Mi, limited: requests.memory=1Gi
 ```
+{{< warning >}}
+To make the resource quota work, two things need to be done.
+
+1.admin set the resource quota limit
+2.User needs to mention the resource quota limit in pod spec.
+
+If the user doesn't specify the resource limit in pod spec, then the resource quota is not enforced on that resource. For example, if the admin set the ephemeral-storage quota and if the user doesn't specify the ephemeral-storage limit in the pod creation spec then the ephemeral-storage resource quota will not be enforced. Thus the user can use n number of ephemeral storage.
```

Comment:
I'm not sure this is technically correct.
Each type/kind of resources are treated in different ways by different controllers. Maybe ephemeral storage is an exception we need to call out, but I don't think resource quota limit need to be mentioned in to Pod `spec` explicitly for all resource types.

Reply:
Hi @tengqm  yes , i have checked with other resources also like secrets and found that resource quota enforce on secrets as 
`error: failed to create secret secrets "db-user" is forbidden: exceeded quota: object-counts, requested: secrets=1, used: secrets=2, limited: secrets=2` . It seems like this is an exception for ephemeral-storage and i have added corresponding changes . Please take a look .
Thanks!!

Reply:
I have modified the PR title for enforcing resource quota limits on ephemeral storage.